### PR TITLE
Update marker position on lat lng change

### PR DESCRIPTION
**Pull request by the-yadu**
I have added an afterUpdate hook to handle lat lng change and update the position of the marker in the mapbox

### CODE REVIEW

File: src/Marker.svelte [hunk 1]
```diff
@@ -35,6 +35,11 @@
 
 	  return () => marker.remove()
 	})
+	
+	afterUpdate(() => {
```
Comment (antony) [inline src/Marker.svelte:39]: I would use a reactive property for this: `$:`

File: src/Marker.svelte [hunk 2]
```diff
@@ -35,6 +35,11 @@
...
+	    // Update marker on map if the lat or lng has been changed
```
Comment (antony) [inline src/Marker.svelte:40]: I have a strict no comments rule across all of my codebases :) It's also pretty obvious what this code does.

**Review by antony — CHANGES_REQUESTED** [review]
Looks good, but there are some minor improvements I think we can make

File: src/Marker.svelte (hunk 1 above)
Comment (the-yadu) [reply src/Marker.svelte]: Done.. Removed it.

File: src/Marker.svelte (hunk 2 above)
Comment (the-yadu) [reply src/Marker.svelte]: Done.. got rid of the hook itself.

**Comment by the-yadu** [comment]
Not that it is any of my business but, Why do you prefer code without comments?

**Comment by antony** [comment]
> Not that it is any of my business but, Why do you prefer code without comments?

Comments are un-testable, and therefore they can be wrong, and nobody will know. Therefore they can't be trusted. If comments can't be trusted, then they are at best, useless, at worst, dangerous.

I prefer well written code which shows intent.

**Comment by antony** [comment]
Added in v6.0.0

**Pull request closed without merging**